test/parallel/test-stream2-readable-legacy-drain.js

```diff
@@ -52,11 +52,4 @@ function drain() {
 
 w.end = common.mustCall();
 
-// Just for kicks, let's mess with the drain count.
-// This verifies that even if it gets negative in the
-// pipe() cleanup function, we'll still function properly.
-r.on('readable', function() {
-  w.emit('drain');
-});
```

Comment:
Why was this removed?

Reply:
> Mixing pipe with 'readable' is a nono since registering readable messes with state.flowing.

Reply:
I'm actually not sure what this is supposed to test and whether is even valid?